axelor-sale/src/main/resources/views/Cart.xml

```diff
@@ -0,0 +1,41 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<object-views xmlns="http://axelor.com/xml/ns/object-views"
+  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://axelor.com/xml/ns/object-views http://axelor.com/xml/ns/object-views/object-views_7.1.xsd">
+
+  <grid name="cart-grid" title="Carts" model="com.axelor.apps.sale.db.Cart">
+    <field name="user"/>
+    <field name="company"/>
+    <field name="partner"/>
+  </grid>
+
+  <form name="cart-form" title="Cart" model="com.axelor.apps.sale.db.Cart"
+    onNew="action-cart-record-default" width="large">
+    <toolbar>
+      <button name="emptyCartBtn" title="Empty cart"
+        onClick="save,action-cart-method-empty-cart"/>
+    </toolbar>
+
+    <panel name="mainPanel">
+      <field name="user" form-view="user-form" grid-view="user-grid"/>
+      <field name="company" form-view="company-form" grid-view="company-grid"/>
+      <field name="partner" form-view="partner-form" grid-view="partner-grid"/>
+      <panel-related field="cartLineList" form-view="cart-line-form"
+        grid-view="cart-line-grid" colSpan="12"/>
+    </panel>
+  </form>
+
+  <action-record name="action-cart-record-default"
+    model="com.axelor.apps.sale.db.Cart">
+    <field name="user" expr="eval:__user__"/>
+    <field name="company" expr="eval:__user__.activeCompany"
+      if="__user__.activeCompany != null"/>
+    <field name="company" expr="eval:__repo__(Company).all().fetchOne()"
+      if="__user__.activeCompany == null &amp;&amp; __repo__(Company).all().count() == 1"/>
+  </action-record>
```

Comment:
Please change it to java method in a service called CartInitValueService. This will be useful for API requests.